tests/rptest/services/producer_swarm.py

```diff
@@ -100,6 +107,9 @@ def start_node(self, node, clean=None):
 
         if self._keys is not None:
             cmd += f" --keys={self._keys}"
+        else:
+            # by default use a very large key space so all partitions are written to
+            cmd += " --keys=18446744073709551557"
```

Comment:
Very cool - largest 64bit prime huh (googled it).

Reply:
I was actually going for largest 63-bit prime but managed to mess that up somehow.